gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,340 @@
+
+[source,ssh]
+----
+flexctl registration renew --help
+----
+
+For Flex Gateway running in a container, invoke `help` as part of the container `run` command:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl mulesoft/flex-gateway registration renew --help
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl docker.io/mulesoft/flex-gateway registration renew --help
+----
+====
+
+Refer to the following topics for more information about renewal command usage:
+
+* <<renew-with-user-password,Renew Registration with User Credentials>>
+* <<renew-with-conn-app,Renew Registration with Connected App Credentials>>
+
+[[renew-with-user-password]]
+=== Renew Registration with User Credentials
+
+Flex Gateway includes the following CLI command to renew registration using Anypoint Platform user credentials:
+
+[source,ssh]
+----
+flexctl registration renew --username=<your-username> --password=<your-password> \
+ <path-to-registration-file>
+----
+
+Replace `<path-to-registration-file>` with the path and filename of the existing `registration.yaml` file. You can optionally output the new registration file to a directory by using the `--output-directory` flag. The `renew` command overwrites the existing registration file if you specify the file's directory as the output directory.
+
+IMPORTANT: Create a new directory called `flex-renew-registration` (or similar) and then run the `renew` command in this new directory. The command creates registration files in this location.
+
+For Flex Gateway running in a container, invoke `renew` as part of the container `run` command:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl \
+-v "$(pwd)":/renew \
+-v <path-to-registration-directory>:/registration \
+-u $UID mulesoft/flex-gateway \
+registration renew \
+--username=<your-username> \
+--password=<your-password> \
+--output-directory=/renew \
+/registration/<registration-filename>
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl --userns=keep-id \
+-v "$(pwd)":/renew:Z \
+-v <path-to-registration-directory>:/registration:Z \
+-u $UID docker.io/mulesoft/flex-gateway \
+registration renew \
+--username=<your-username> \
+--password=<your-password> \
+--output-directory=/renew \
+/registration/<registration-filename>
+----
+====
+
+Invoking the command results in the following output:
+
+[source,text]
+----
+Starting registration renewal, please be patient.
+Registration renewal completed, the configuration files were written in directory ".".
+For security, modify the file permissions to restrict production scenario access to the user running flex.
+
+Side effect: the following files were written in the current directory
+registration.yaml
+----
+
+[[renew-with-conn-app]]
+=== Renew Registration with Connected App Credentials
+
+Flex Gateway includes the following CLI command to renew registration using connected app credentials:
```

Comment:
To..., do...

Reply:
This text was removed in the whole task-ification effort.